recipes/MADL-CPU-IntelMPI/docker/Dockerfile

```diff
@@ -0,0 +1,59 @@
+#Dockerfile for MDL (Microsoft Distributed Learners)
+
+FROM ubuntu:16.04
+MAINTAINER Saeed Maleki Todd Mytkowicz Madan Musuvathi Dany rouhana <https://github.com/Azure/batch-shipyard>
+ENV DEBIAN_FRONTEND=noninteractive
+
+#install base system
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		build-essential \
+		apt-utils \
+        	openssh-client \
+        	openssh-server \
+        	iproute2 \
+		gcc \
+		emacs \
+		iotop \
+		gfortran \
+		libopenblas-dev \
+		libatlas-base-dev \
+		liblapacke-dev \
+		cpio \
+		openmpi-bin \
+		openmpi-common \
+		libopenmpi-dev && \
+		apt-get clean && \
+		rm -rf /var/lib/apt/lists/* 
+
+# configure ssh server and keys
+RUN mkdir -p /root/.ssh && \
+    echo "Host *\n\tPort 23\n\tStrictHostKeyChecking no\n\tUserKnownHostsFile /dev/null" > /root/.ssh/config && \
+    mkdir /var/run/sshd && \
+    ssh-keygen -A && \
+    sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/' /etc/ssh/sshd_config && \
+    sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd && \
+    ssh-keygen -f /root/.ssh/id_rsa -t rsa -N '' && \
+    chmod 600 /root/.ssh/config && \
+    chmod 700 /root/.ssh && \
+    cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys	
+
+#set parasail dir	
+WORKDIR /parasail	
+
+# remove intel components 
+RUN rm -rf /opt/intel
```

Comment:
Remove this, not needed for openmpi

Reply:
I thought I did.  I cannot see it in the version I have opened.